<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWidget</class>
 <widget class="QWidget" name="RegWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PushButton_RegAck">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>510</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/windows/Reg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_BackToLogin">
   <property name="geometry">
    <rect>
     <x>913</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/windows/close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_RID">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lable_RPassword_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_RPasswordAck">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>确认密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_RID">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_RPassword">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_RPasswordAck">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Name">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>昵称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Name">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>641</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/windows/LW.jpg);</string>
   </property>
  </widget>
  <zorder>listView</zorder>
  <zorder>PushButton_RegAck</zorder>
  <zorder>PushButton_BackToLogin</zorder>
  <zorder>Label_RID</zorder>
  <zorder>Lable_RPassword_2</zorder>
  <zorder>Label_RPasswordAck</zorder>
  <zorder>LineEdit_RID</zorder>
  <zorder>LineEdit_RPassword</zorder>
  <zorder>LineEdit_RPasswordAck</zorder>
  <zorder>Label_Name</zorder>
  <zorder>LineEdit_Name</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
